internal/builders/docker/pkg/builder.go

```diff
@@ -481,25 +481,36 @@ func (c *GitClient) checkoutGitCommit() error {
 func saveToTempFile(verbose bool, readers ...io.Reader) ([]string, error) {
 	var files []string
 	for _, reader := range readers {
-		bytes, err := io.ReadAll(reader)
-		if err != nil {
-			return files, err
+		var allBytes []byte
+		if verbose {
+			fmt.Print("\n\n>>>>>>>>>>>>>> output from command <<<<<<<<<<<<<<\n")
 		}
 
-		if verbose {
-			if len(bytes) > 0 {
-				fmt.Print("\n\n>>>>>>>>>>>>>> output from command <<<<<<<<<<<<<<\n")
-				fmt.Printf("%s", bytes)
-				fmt.Print("=================================================\n\n\n")
+		scanner := bufio.NewScanner(reader)
+		for scanner.Scan() {
```

Comment:
I think this will still try to process each provided Reader to completion before moving to the next. So we'd at least get the output from `stdout`, but if the process is terminated, we'd still never get to see `stderr`. 

One possible solution is to launch a goroutine to process each Reader concurrently. My golang is very out of date, so I'm not sure about the best practices around doing something like this these days.

Reply:
(Updated this to try to concurrently print/process each provided file).